### ISSUE TITLE
SVG support

### ISSUE THREAD

**Issue by jmatsushita** [paf31/purescript-drawing#22]
As mentioned in sharkdp/purescript-flare#13 a `purescript-drawing` SVG backend would be quite useful!

**Comment by paf31** [comment]
Yes, agreed! I suppose we'd want a `purescript-svg` library first.

**Comment by jmatsushita** [comment]
Maybe something @ryanartecona would be interested in, given past mentions of a `purescript-halogen-svg` slamdata/purescript-halogen#133 ?

**Comment by ryanartecona** [comment]
@jmatsushita Thanks for thinking of me! These days I've moved into more of an infrastructure/opsy role at work, and I haven't done any frontend work for some time. I'd still reach for PureScript if I have cause in the future, but probably even better for my mental health would be to avoid frontend work entirely. :)

**Comment by cuongngosa** [comment]
Any update ?

**Comment by paf31** [comment]
I don't think anyone's working on this.
